### DISCUSSION TITLE
Are there any showcase using this package?

### DISCUSSION THREAD

**Discussion by sawirricardo** [saloonphp/saloon#55] [Q&A]
It will be great to see what the community have built with this package.

**Comment by Sammyjo20** [comment]
Absolutely @sawirricardo! I saw that you moved your SDK to saloon and it looks good, what did you think of it?

**Reply by sawirricardo** [reply]
For me this package helps me to standardize the way I interact with external APIs (especially when they don't have dedicated SDK), really helpful and align with the 'conventions over configurations'. and this package helps me to be more productive with my projects. Nice work, @Sammyjo20

**Reply by thyseus** [reply]
https://docs.saloon.dev/conclusion/showcase
